docs/cheatsheet/index.rst

```diff
@@ -1,33 +1,33 @@
 .. eql:section-intro-page:: cheatsheet
 
-===========
-Cheatsheets
-===========
+==============================
+Cheatsheets: EdgeDB by example
+==============================
 
-A collection of cheatsheets.
+Just getting started? Keep an eye on this collection of cheatsheets with handy examples for what you'll need to get started with EdgeDB. After familiarizing yourself with them, feel free to dive into more EdgeDB via our longer [interactive tutorial](https://tutorial.edgedb.com/) and **much** longer [Easy EdgeDB textbook](https://website-git-myst.edgedb.vercel.app/easy-edgedb).
```

Comment:
As I suggested in other PR, please reformat to have max 79 characters per line -- that's a linter rule that won't let tests pass.